Documentation/contributing/testing/ci.rst

```diff
@@ -153,103 +153,67 @@ packet.net before the branch is merged.
 Testing matrix
 ^^^^^^^^^^^^^^
 
-We are currently testing following kernel - k8s version pairs in our CI:
-
-+--------------------+------------------+
-| Kubernetes version | Kernel version   |
-+====================+==================+
-| Vagrant k8s clusters per PR           |
-+--------------------+------------------+
-| 1.13               | 5.x.x (net-next) |
-+--------------------+------------------+
-| 1.19               | 4.19.57          |
-+--------------------+------------------+
-| 1.20               | 4.9              |
-+--------------------+------------------+
-| Vagrant k8s clusters per backport     |
-| (in addition to PR)                   |
-+--------------------+------------------+
-| 1.{13-19}          | 4.9              |
-+--------------------+------------------+
-| GKE clusters                          |
-+--------------------+------------------+
-| 1.15.12            | 4.19.112+        |
-+--------------------+------------------+
+Up to date CI testing information regarding k8s - kernel version pairs can
+always be found in the `Cilium CI matrix`_.
 
-.. _trigger_phrases:
+.. _Cilium CI matrix: https://docs.google.com/spreadsheets/d/1TThkqvVZxaqLR-Ela4ZrcJ0lrTJByCqrbdCjnI32_X0
 
+.. _trigger_phrases:
 
 Triggering Pull-Request Builds With Jenkins
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Note: depending on the time at which it was built, the documentation below might
+be outdated. Up to date required status and trigger phrases can always be found
+in the `Cilium CI matrix`_.
+
```

Comment:
Is there a reason to treat this and table differently from the other table we're removing?

I don't think we should ever commit something that says "this documentation may be deliberately out of date", that just shows we don't care enough about the docs to keep it correct (which is a process problem). If we won't commit to keeping it up to date, we shouldn't document it.

Reply:
I'll quote Paul above, which was thinking the same thing as I did initially:
> I think keeping the docs for trigger phrases below makes sense because they should change less often (compared to Kubernetes versions).

I was also taking advantage of the text format to keep some contextual paragraphs that I think were better suited there rather than in the CI matrix directly.

With that said, the "caring about docs" argument sounds quite compelling to me. Since we are moving to having triggers directly displayed in PR status names, I expect most people not to care anymore about individual triggers documentation (these will be obvious and always visible). However aggregate triggers still need to be kept and explained somewhere.

So I have 3 proposals:
- Move contextual paragraphs from docs to CI matrix (but may be a bit cluttered?), remove whole triggers table
- Keep contextual paragraphs in docs, remove whole triggers table
- Keep contextual paragraphs in docs, keep aggregate triggers, remove individual triggers (the reasoning being aggregate will most likely never change over time)

What do you think would work best?

Reply:
The third option seems reasonable to me, agree that individual triggers will change given there's kernel / k8s version numbers in there, but the aggregate triggers should stay the same.

Reply:
Modified in latest push, awaiting your validation ;)

Reply:
(repushed for rebase to get PR checks to work + adding "retrigger" to words whitelist)

Reply:
LGTM :+1: